src/sentry/integrations/vsts/issues.py

```diff
@@ -81,6 +81,7 @@ def create_issue(self, data, **kwargs):
         return {
             'key': created_item['id'],
             # 'url': created_item['_links']['html']['href'],
```

Comment:
Do we wanna remove this?

Reply:
Sure I can follow up with another PR. That's a bit less important and this needs to get out quickly. But this was from the original vsts plugin and I just copied over as I wasn't sure if we were going to use it.